corehq/apps/zapier/views.py

```diff
@@ -69,7 +70,8 @@ def post(self, request, domain, *args, **kwargs):
         else:
             return HttpResponse(status=400)
 
-        return HttpResponse('OK')
+        # respond with the ID so that zapier can use it to unsubscribe
+        return json_response({'id': subscription.id})
```

Comment:
Actually, I guess ours does use a json handler that calls `to_json()` on an object if possible, so I'll keep it for now.